### DISCUSSION TITLE
Finding unused views?

### DISCUSSION THREAD

**Discussion by ronag** [apache/couchdb#3756] [Q&A]
Is there any form of metrics telling when a view was last accessed that can be used to find unused views? Since couch 2+ actively updates all views regardless whether they are accessed or not I think it becomes more important to do cleanups.

**Comment by big-r81** [comment]
Hi, you have to check your access logs to find out which routes are used. [See](https://couchdb.slack.com/archives/C49LEE7NW/p1649674113014149)
